justfile

```diff
@@ -42,6 +43,8 @@ test:
 	cargo run --package runner -- run --config {{qemu_virt}} --firmware os_ecall
 	cargo run --package runner -- run --config {{qemu_virt_release}} --firmware default
 
+	cargo run --package runner -- run --config {{qemu_virt_hello_world_payload}} --firmware opensbi-jump
+
```

Comment:
Let's move that one just bellow `--firmware opensbi` to keep things grouped (it's a test with an external project).